index.html

```diff
@@ -17,15 +17,17 @@
                 var table = jsonData["valueRanges"];
                 var nameData = table[0]["values"];
                 var dateData = table[1]["values"];
+                var runtimeData = table[2]["values];
                 var selection = Math.floor(Math.random() * (nameData.length - 1)) + 1;
                 console.log('Index: ' + selection + ' Title Selected: ' + nameData[selection][0] + ' Date: ' +
-                    dateData[selection][0]);
+                    dateData[selection][0] + ' Runtime: ' + runtimeData[selection][0]);
                 $('#movieBox').find("h1").text(nameData[selection]);
                 $('#movieBox').find("h2").text('('+ dateData[selection] + ')');
+                $('#movieBox').find("h3").text(runtimeData[selection] + 'minutes');
```

Comment:
Formatting will not be the same as the text in the header. I suggest `"(" + runtimeData[selection] + " minutes )");`